Makefile

```diff
@@ -44,7 +44,7 @@ requre:  ## Regenerate test data for integration tests
 ## Documentation
 ##
 docs: clean  ## Build documentation
-	hatch run docs:html
+	sphinx-build -b html docs/ docs/_build
```

Comment:
Shouldn't this come with some kind of "install devel (or docs) environment" target? `hatch` ran the command in the `docs` environment, without `hatch` - or some kind of `uv` replacement - this would run outside of a virtual environment, and I'm pretty sure I don't have `sphinx-build` installed anywhere on my workstation. Maybe some target running just `uv sync --locked --extra docs` as a depedency of `docs`?